inference/__main__.py

```diff
@@ -22,12 +52,19 @@ def main():
         type=str,
         help="Prefix for the output directory"
     )
-    
+
+    args.add_argument('--use-local', action="store_true", help="Use this flag when running from local prompts.")
+
+    # Reuired when use local is passed
+    args.add_argument(
+        "--dataset", type=str, required="--use-local" in sys.argv, help="The local dataset in JSON format to get from this computer."
+    )
+    # Only required when use local is not passed
     args.add_argument(
-        "--lang", type=str, required=True, help="Target language for completions"
+        "--lang", type=str, required="--use-local" not in sys.argv, help="Target language for completions"
```

Comment:
wow, I have never seen this before. Great idea, I think.

Reply:
Thanks! I stole it from stack overflow a while ago and have been using it in a few personal projects

Reply:
Essentially this allows for me to set which arguments are required, so when someone wants to use normal humaneval they can, but if someone wants to use a local set of prompts it doesn't require them to put in the --lang parameter